chains/boba/metadata.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.routescan.io/v2/network/mainnet/evm/288/etherscan/api
+    family: routescan
+    name: bobascan
+    url: https://bobascan.com
+chainId: 288
+displayName: Boba Mainnet
+domainId: 288
+isTestnet: false
+name: boba
+nativeToken:
+  decimals: 18
+  name: Ether
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: https://mainnet.boba.network
```

Comment:
Here the `deployer` information is missing as an example you can reference one of Hyperlane deployed [chains definitions](https://github.com/hyperlane-xyz/hyperlane-registry/blob/main/chains/arbitrum/metadata.yaml)

Reply:
Fixed!